test/livebook/sessions_test.exs

```diff
@@ -41,6 +41,15 @@ defmodule Livebook.SessionsTest do
 
       Session.close(session.pid)
     end
+
+    test "returns an error if session comes from a different boot" do
+      Application.put_env(:livebook, :random_boot_id, "aaa")
+      id = Livebook.Utils.random_node_aware_id()
+
+      assert Sessions.fetch_session(id) == {:error, :not_found}
+      Application.put_env(:livebook, :random_boot_id, "bbb")
```

Comment:
If the test changes the value, then existing ids become "invalid", so ideally we would revert, right?

Reply:
Theoretically no, we only check it in case we don't find it. :)

Reply:
Ah I see, ok!